IdentityCore/src/requests/broker/MSIDBrokerTokenRequest.m

```diff
@@ -186,7 +186,9 @@ - (NSDictionary *)defaultPayloadContents:(NSError **)error
         queryDictionary[MSID_NESTED_AUTH_BROKER_CLIENT_ID] = self.requestParameters.nestedAuthBrokerClientId;
         queryDictionary[MSID_NESTED_AUTH_BROKER_REDIRECT_URI] = self.requestParameters.nestedAuthBrokerRedirectUri;
     }
-
+    
+    [queryDictionary msidSetNonEmptyString:self.requestParameters.clientSku forKey:MSID_CLIENT_SKU_KEY];
```

Comment:
where/how client sku used for skipping account validation ?

Reply:
Setting client sku here: https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/1993/files#diff-558fd2bfadacab5121f3004397dd79c8e32c3e6c83e0d6cdc265ef814f6f1c9bR1209

This is also being used by MSAL CPP.

Reply:
This parameter has been added for future purposes if we need to do some specific task based on calling SDK.

Reply:
aah ok, thanks